### ISSUE TITLE
Package publishing from AppVeyor broken

### ISSUE THREAD

**Issue by martincostello** [aspnet-contrib/AspNet.Security.OAuth.Providers#338]
After merging #335 into the `dev-3.0.0` branch I noticed that the packages weren't being uploaded to MyGet anymore.

This is because I had to add an [`appveyor.yml`](https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers/blob/dev-3.0.0/appveyor.yml) so it would build correctly, which has overridden whatever was set up in AppVeyor CI in the admin section.

@PinpointTownes, could you submit a PR to add the appropriate YAML to the file on the `dev-3.0.0` branch with the encrypted NuGet API key(s) so that this starts working again? 😃 

This'll also affect https://github.com/aspnet-contrib/AspNet.Security.OpenId.Providers/pull/63.

**Comment by kevinchalet** [comment]
I'm currently OOO, but here's the part you need if you want to fix that before I come back home 😄 

```
deploy:
- provider: NuGet
  server: https://www.myget.org/F/aspnet-contrib/
  api_key:
    secure: 0D9BDh7mkEuJ7sQTkI+L7KhntokQjMRw7UjucOl67mWKIeg2m249Vvc1bbyqDDrj
  skip_symbols: false
  symbol_server: https://www.myget.org/F/aspnet-contrib/
```

It's the same key for both repos.

**Comment by martincostello** [comment]
No problem, I'll sort that out then! Thanks!

**Comment by martincostello** [comment]
Resolved by d2e093b35260af38bcab9e93dd9da44e20d49d8b.
